<?xml version="1.0"?>
<launch>
	<include file="$(find brl_teleop_phantom)/launch/phantom_slave.launch" />
	<node name="SMSS_slave" pkg="brl_teleop_controller" type="phantom_single_controller" output="screen">
		<param name="popc_enable" value="true"	/>

		<param name="design_velocity_src_name" value="/SMSS_master" />
    	<param name="design_velocity_topic_name" value="velocity" />

		<param name="actual_velocity_src_name" value="/phantom_slave" />
    	<param name="actual_velocity_topic_name" value="velocity" />
		
		<rosparam param="gains">{'P': 0.1, 'D': 1}</rosparam>

	</node>
</launch>
